dashboard/test/models/regional_partner_test.rb

```diff
@@ -65,4 +65,59 @@ class RegionalPartnerTest < ActiveSupport::TestCase
       regional_partner.program_manager = program_manager.id
     end
   end
+
+  test 'find_by_region finds matching regional_partner' do
+    # partners and their mappings
+
+    # regional_partner_WA
+    # regional_partner_WA_98105
+    # regional_partner_98105
+    # regional_partner_NY
+    regional_partner_ny = create :regional_partner, name: "partner_NY"
+    regional_partner_ny.mappings.find_or_create_by(state: "NY")
+
+    # regional_partner_MA_02138
+    regional_partner_ma_02138 = create :regional_partner, name: "partner_MA_02138"
+    regional_partner_ma_02138.mappings.find_or_create_by(state: "MA")
+    regional_partner_ma_02138.mappings.find_or_create_by(zip_code: "02138")
+
+    # regional_partner_CA_94305
+    regional_partner_ca_94305 = create :regional_partner, name: "partner_CA_94305"
+    regional_partner_ca_94305.mappings.find_or_create_by(state: "CA")
+    regional_partner_ca_94305.mappings.find_or_create_by(zip_code: "94305")
+
+    # regional_partner_70808
+    regional_partner_70808 = create :regional_partner, name: "partner_70808"
+    regional_partner_70808.mappings.find_or_create_by(zip_code: "70808")
+
+    # regional_partner_no_mappings
```

Comment:
This too. The test seems incomplete, or the comments are extra. Also I don't think the comments above for each  variable declaration add any value.